jsp/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+#
+#  Copyright 2008 Search Solution Corporation
+#  Copyright 2016 CUBRID Corporation
+# 
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+# 
+#       http://www.apache.org/licenses/LICENSE-2.0
+# 
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+# 
+#
+include(FindJava)
+find_package(Java 1.5 COMPONENTS Development)
```

Comment:
How about upgrade java version from 1.5 to 1.6 ?
If upgrading, you need to check upgraded jni version.

Reply:
I upgraded the java version number to find_package at 787d05a